src/formats/PDB.cpp

```diff
@@ -337,47 +339,47 @@ void PDBFormat::read_ATOM(Frame& frame, const std::string& line,
         atom.set_type(trim(line.substr(76, 2)));
     }
 
-    auto altloc = line.substr(16,1);
+    auto altloc = line.substr(16, 1);
     if (altloc != " ") {
         atom.set("altloc", altloc);
     }
 
     try {
-        auto x = std::stod(line.substr(31, 8));
-        auto y = std::stod(line.substr(38, 8));
-        auto z = std::stod(line.substr(46, 8));
+        auto x = parse<double>(trim(line.substr(31, 8)));
```

Comment:
Trimming is now required due to the length check, correct?

Reply:
Yep!